otel/conf.go

```diff
@@ -421,11 +421,16 @@ windowsperfcounters/mssql_{{.MSSQLID}}:
         action: update
         new_name: processes/cpu_time
         operations:
+          - action: experimental_scale_value
+            experimental_scale: 1000000
           # change data type from double -> int64
           - action: toggle_scalar_data_type
           - action: add_label
             new_label: process
             new_value: all
+          - action: delete_label_value
```

Comment:
Are you sure that `delete_label_value` is the correct action and this shouldn't be merged with one of the other CPU usage metrics, or left separate?

Reply:
yes, i am sure.

Reply:
The reason is after checking the value emit by otel, it's getting the right value from `/proc/pid/stat`'s `utime` and `stime`. Therefore, the wait value doesn't fit anywhere, it shall just be dropped. See b/191062380#comment14